lib/ansible/plugins/inventory/netbox.py

```diff
@@ -39,6 +39,11 @@
                   region, site, role, platform, and/or tenant. Please check official netbox docs for more info.
             default: False
             type: boolean
+        interfaces:
+            description:
+                - If True, it adds dcim interface information in host vars.
+            default: False
+            type: boolean
```

Comment:
please add a `version_added: "2.8"`

Reply:
Thanks for the review.  Apologies for the subsequent commit mess